component/file_cache/file_cache.go

```diff
@@ -160,6 +175,7 @@ func (c *FileCache) Priority() internal.ComponentPriority {
 //	this shall not block the call otherwise pipeline will not start
 func (c *FileCache) Start(ctx context.Context) error {
 	log.Trace("Starting component : %s", c.Name())
+	//c.closeSignal = make(chan int)
```

Comment:
I had that there to help indicate to the async thread whether or not it could stop running. I ran into some issues with it because it was ending the thread early, so I just commented it out with the plan to revisit it at a later time

Reply:
I just rely on the go compilier to cleanup running threads when we end the program, but I'm not sure if that's something we want in the long term